src/Command/AbstractCommand.php

```diff
@@ -208,6 +208,7 @@ public function batchInsert(string $table, array $columns, iterable $rows): stat
 
     public function insertBatch(string $table, iterable $rows, array $columns = []): static
     {
+        //mark this as deprecated and remove in future versions
```

Comment:
Shouldn't it be done? i.e. phpdoc/annotation to add.

Reply:
We are in discussion stage. No clear vision of how it will be implemented yet. Thus it is here for a while. After we’ll come to an agreement, I will format the code and docs properly :)

Reply:
We don't can use several `INSERT` commands in one SQL statement, so we don't can resolve our task with current `insertBatch` method.

May be replace `CommandCollection` class to `InsertBatchCommandCollection` and return in `CommandInterface::insertBatch()` method this collection?